ilc/client/ClientRouter.js

```diff
@@ -134,6 +137,12 @@ export default class ClientRouter {
                 },
             });
         }
+
+        const processedUrl = this.#urlProcessor.process(this.#currentUrl);
+        if (this.#currentUrl !== processedUrl) {
+            console.info(`ILC: Redirecting to "${processedUrl}"...`);
+            this.#location.replace(processedUrl);
```

Comment:
Need to discuss it
Because I am not sure about it

Reply:
Discussed
I removed it from here
Added to patched update state require of a function that would return a created instance of URL processor